### ISSUE TITLE
Tippy error

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#2400]
I updated to the latest pre-release packages and encountering an issue with Tippy, which prevents the page from rendering. 

Note that I've also updated React v16.3.9 and React.dom v16.0.5 to use v16.8.0. That is, in an attempt to fix this issue.

<img width="1182" alt="Screen Shot 2019-06-28 at 3 37 58 PM" src="https://user-images.githubusercontent.com/17481322/60366878-c9326280-99ba-11e9-8303-05607a875118.png">

<img width="267" alt="Screen Shot 2019-06-28 at 3 30 25 PM" src="https://user-images.githubusercontent.com/17481322/60366498-b0757d00-99b9-11e9-8007-913303ce595f.png">

**Comment by dlabrecq** [comment]
A couple notes from my discussion with @redallen .

Zack mentioned that Tippy requires a function in a higher version of React. (I don't believe the versions in PF and Cost Management have changed in quite a while.) Regardless, the problem wasn't resolved by updating React and React.dom to v16.8.0 for Cost Management.

Zack also suggested that "since there's both PF3 and PF4 and different versions of react, i think it's probably this:" https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react (edited) 

If either is true, that breaks the PatternFly migration story. It appears that my only alternative is to remove PF3 from Cost Management?

**Comment by dlabrecq** [comment]
Added a dependency for react v16.8.6, which seems to be helping. I'm now seeing issues #2403 and #2400.

Unfortunately, our tests are failing due to lodash-es, which appears to be used by react-table? Cost Management is not using this.

<img width="881" alt="Screen Shot 2019-06-30 at 4 09 46 PM" src="https://user-images.githubusercontent.com/17481322/60401585-d170d600-9b51-11e9-988a-66bba9fdbb43.png">

**Comment by rachael-phillips** [comment]
@seanforyou23 #2395 solve @dlabrecq 's issue? @tlabaj making you aware

**Comment by seanforyou23** [comment]
@rachael-phillips not as is, but I'm looking to see if there's a way to extend that PR to fix it. Haven't been able to reproduce it locally yet - will keep trying. Working on this today.

**Comment by redallen** [comment]
@dlabrecq On the package side, someone else had a problem with lodash imports in react-charts in #2222 and it was solved by making `lodash-es` a dependency of react-charts instead of peerDependency. If you use `lodash-es` elsewhere I'd make sure all those versions align. (But koko-ui looks fine).
```
➜  koku-ui git:(master) ✗ npm ls lodash-es
koku-ui@0.0.1 /home/redallen/src/koku-ui
├─┬ @patternfly/react-table@2.13.27
│ └── lodash-es@4.17.11 
└─┬ @red-hat-insights/insights-frontend-components@3.41.68
  ├─┬ react-redux@5.0.7
  │ └── lodash-es@4.17.10 
  └─┬ redux@3.7.2
    └── lodash-es@4.17.11
```

On the Jest side, it looks like it can't handle the module syntax in `lodash-es`, so perhaps you have to tell Jest to transpile those files in `node_modules` with however you have its babel config set up.

**Comment by dlabrecq** [comment]
FYI, @ patternfly/react-core still has the following dependencies. Although, it appears that 16.8.6 is now required?
```
react-dom: "^15.6.2 || ^16.4.0"
react: "^16.4.0"
```

**Comment by dlabrecq** [comment]
Thanks, @redallen. We don't use lodash-es in Cost Management and prefer not to make this a dependency just for a test.

We may end up deleting this test rather than transpile lodash-es.

**Comment by redallen** [comment]
@dlabrecq Yes, `16.8.6` is now required for both `react` and `react-dom` because of the Tippy upgrade. I've opened #2408 for this. I cut a branch to that Friday without issue. There is the small implementation detail that consumers will have to upgrade React as well in order to use Tippy components.

**Comment by dlabrecq** [comment]
Looks like Tippy is having an affect on the chart legends in Cost Management as well. Haven't narrowed it down yet, but we're adding the tooltips to our legends -- the text is not rendering.

![Screen Shot 2019-07-01 at 10 40 38 AM](https://user-images.githubusercontent.com/17481322/60446153-c28c3100-9bee-11e9-9394-8542c5a042d2.png)

**Comment by tlabaj** [comment]
Closed #2365 since this issue address it.

**Comment by dlabrecq** [comment]
I'm still seeing an error with Tippy. Does this mean I have to downgrade my version of React now?
